packages/server/tests/acceptance/conformityTests.spec.ts

```diff
@@ -0,0 +1,295 @@
+  gasPrice,
+  gasLimit: gasLimit,
+  type: 1,
+};
+
+let transaction1559 = {
+  chainId: 0x12a,
+  to: receiveAccountAddress,
+  from: sendAccountAddress,
+  value,
+  gasPrice,
+  maxPriorityFeePerGas: gasPrice,
+  maxFeePerGas: gasPrice,
+  gasLimit: gasLimit,
+  type: 2,
+};
+
+let createContractLegacyTransaction = {
+  chainId: 0x12a,
+  to: null,
+  from: sendAccountAddress,
+  gasLimit: gasLimit,
+  gasPrice: gasPrice,
+  type: 0,
+  data: bytecode,
+};
+
+async function getTransactionCount() {
+  const request = {
+    jsonrpc: '2.0',
+    id: 1,
+    method: 'eth_getTransactionCount',
+    params: [sendAccountAddress, 'latest'],
+  };
+
+  const response = await sendRequestToRelay(request);
+
+  return response.result;
+}
+
+async function getLatestBlockHash() {
+  const request = {
+    jsonrpc: '2.0',
+    method: 'eth_getBlockByNumber',
+    params: ['latest', false],
+    id: 0,
+  };
+
+  const response = await sendRequestToRelay(request);
+
+  return response.result.hash;
+}
+
+function splitReqAndRes(content) {
+  /**
+   * Splits a given input string into distinct segments representing the request and the response.
+   *
+   * @param {string} content - The input string to be segmented.
+   * @returns {{ request: string, response: string }} - An object containing the separated request and response strings.
+   */
+  const lines = content.split('\n');
+  const filteredLines = lines.filter((line) => line != '').map((line) => line.slice(3));
+
+  return { request: filteredLines[0], response: filteredLines[1] };
+}
+
+async function sendRequestToRelay(request) {
+  try {
+    const response = await axios.post(relayUrl, request);
+    return response.data;
+  } catch (error) {
+    console.error(error);
+    throw error;
+  }
+}
+
+async function signAndSendRawTransaction(transaction) {
+  transaction.nonce = parseInt(await getTransactionCount(), 16);
+  const signed = await signTransaction(transaction, localNodeAccountPrivateKey);
+  const request = {
+    jsonrpc: '2.0',
+    id: 1,
+    method: 'eth_sendRawTransaction',
+    params: [signed],
+  };
+
+  const response = await sendRequestToRelay(request);
+  const requestTransactionReceipt = {
+    id: 'test_id',
+    jsonrpc: '2.0',
+    method: 'eth_getTransactionReceipt',
+    params: [response.result],
+  };
+  const transactionReceipt = await sendRequestToRelay(requestTransactionReceipt);
+  return {
+    transactionHash: response.result,
+    blockHash: transactionReceipt.result.blockHash,
+    transactionIndex: transactionReceipt.result.transactionIndex,
+    blockNumber: transactionReceipt.result.blockNumber,
+  };
+}
+
+async function checkRequestBody(fileName, request) {
+  /**
+   * Modifies a request object for compatability with our relay.
+   *
+   * @param {string} fileName - The name of the file associated with the request.
+   * @param {Object} request - The request object to be modified.
+   * @returns {Object} - The modified request object.
+   */
+  if (
```

Comment:
nit: The check values for `request.params[0]` seem random, can you assign then to a constant with an informative variable name so the test is more understandable.

Also might be better to group the `eth_sendRawTransaction` considerations